modules/grafana/Makefile

```diff
@@ -0,0 +1,43 @@
+# You might want to set these when you run make
+GRAFANA_ADMIN_PASSWORD ?= admin
+LOCAL_DASHBOARD_DIRECTORY ?= dashboards
+
+# You should leave these as the default values
+GRAFANA_LOCAL_DOCKER_NAME = grafana_local
+GRAFANA_SYNC_DOCKER_NAME = grafana_sync
+CONTAINER_DASHBOARD_DIRECTORY = /app/dashboards
+TMP_GITLAB_REPO_DIRECTORY = /tmp/build-harness-extentions-private
+
+.PHONY: grafana/develop
+# Start up a local grafana instance with live datasources, and sync to dashboards on your local disk. Only for mintel internal access, try grafana/develop-opensource for general use.
+grafana/develop: grafana/setup-local-grafana-mintel grafana/setup-grafana-syncer grafana/cleanup
+
+.PHONY: grafana/develop-opensource
+# Start up a local grafana instance with no datasources configured, and sync to dashboards on your local disk
+grafana/develop-opensource: grafana/setup-local-grafana-opensource grafana/setup-grafana-syncer grafana/cleanup
+
+grafana/private:
+	@git clone git@gitlab.com:mintel/satoshi/tools/build-harness-extentions-private.git ${TMP_GITLAB_REPO_DIRECTORY}
```

Comment:
It might also be useful to have the option to pass in the branch here.

```
BUILD_HARNESS_EXTENSIONS_PRIVATE_BRANCH?=main
@git clone git@gitlab.com:mintel/satoshi/tools/build-harness-extentions-private.git -b $(BUILD_HARNESS_EXTENSIONS_PRIVATE_BRANCH) ${TMP_GITLAB_REPO_DIRECTORY}

Reply:
Done in new revision